Sources/SwifterSwift/CoreGraphics/CGSizeExtensions.swift

```diff
@@ -17,6 +17,21 @@ import UIKit
 import Cocoa
 #endif
 
+// MARK: - Methods
+public extension CGSize {
+
+    /// SwifterSwift: Returns the aspect ratio.
+    var aspectRatio: CGFloat {
+        return height == 0 ? 0 : width / height
+    }
+
+    /// SwifterSwift: Returns width or height, whatever is the bigger value.
+    var max: CGFloat {
```

Comment:
Not sure about this tho, because calling that don't seem readable. Example `size.max` is the max size? I mean I couldn't tell just looking to the property or a call of that what it does. Don't know, let me know if you think it makes sense :)

Reply:
I agree. Do you think _maxDimension_ would be a better name? If we want to avoid _max_, we could name it _largerDimension_.

Reply:
`maxDimension` sounds better to me :))

Reply:
If you're going to add `maxDimension`, you might as well add `minDimension` for parity.

Reply:
Done :)